src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -793,7 +796,9 @@ def _stringify_generator_output(self, outputs: dict):
 
     def _submit_to_scheduler(self, context: FlowExecutionContext, inputs, nodes: List[Node]) -> Tuple[dict, dict]:
         if not isinstance(self._node_concurrency, int):
-            raise NodeConcurrencyNotFound("Need to set node concurrency as using flow executor.")
+            raise NodeConcurrencyNotFound(
+                message_format=f"Need to set valid node concurrency to run, current value is {self._node_concurrency}."
```

Comment:
How does the user handle this error? We should provide a troubleshooting guide in the error message.
i.e. We can tell the user what are valid node concurrency values.

Reply:
It's one System error. Customer usually can't do anything for it.